pandas/core/window.py

```diff
@@ -181,6 +181,10 @@ def __unicode__(self):
         return "{klass} [{attrs}]".format(klass=self._window_type,
                                           attrs=','.join(attrs))
 
+    def __iter__(self):
+        url = 'https://github.com/pandas-dev/pandas/issues/11704'
+        raise NotImplementedError('See issue #11704 %s' % (url,))
```

Comment:
Can you use `.format` here instead? Alternately could get away with just referencing the issue number

Reply:
@WillAyd Can you point me to the file where the test should be?